sdk/include/ndk/iotypes.h

```diff
@@ -935,6 +935,9 @@ typedef struct _EXTENDED_DRIVER_EXTENSION
     UNICODE_STRING ServiceKeyName;
     PIO_CLIENT_EXTENSION ClientDriverExtension;
     PFS_FILTER_CALLBACKS FsFilterCallbacks;
+    PVOID KseCallbacks;
```

Comment:
If you know anything about which Windows versions these fields are present in (I seem them in Win10 symbols but not in Win7 symbols. Not sure about in between), that could be useful in a comment.

Reply:
or within a `#ifdef NTDDK_xxx`

Reply:
kse/dv since 8, verifier since 8.1
Will add a comment
http://terminus.rewolf.pl/terminus/structures/ntdll/_DRIVER_EXTENSION_x86.html